tensorboard/webapp/plugins/npmi/util/filter_annotations.ts

```diff
@@ -91,8 +91,11 @@ function checkValuesPassMetricArithmetic(
     }
     return valueDataElements.some((valueDataElement) => {
       if (valueDataElement.metric === stripMetricString(element.metric)) {
-        if (!valueDataElement.nPMIValue && metricFilter.includeNaN) {
-          return true;
+        if (valueDataElement.nPMIValue === null) {
+          if (metricFilter.includeNaN) {
+            return true;
+          }
+          return false;
```

Comment:
`return metricFilter.includeNaN`?

Reply:
Oh, true. Thank you. Can you merge the PR for me? I dont have the rights anymore.